src/content/app/entity-viewer/variant-view/population-allele-frequencies/PopulationAlleleFrequencies.tsx

```diff
@@ -195,8 +288,9 @@ const PopulationFrequenciesTable = (props: {
       <thead>
         <tr>
           <ColumnHead>Population</ColumnHead>
+          <ColumnHead></ColumnHead>
           <ColumnHead>
-            <span className={styles.alleleColumnTitle}>Allele</span>
+            <span className={styles.alleleColumnTitle}>Alt Allele</span>
```

Comment:
Compare with the population frequency table currently. It used to say "allele". You can select the reference allele and look at its frequency as well. Why does the label say "alt"?

Check with Variation to make sure.